gradle/dependencies.gradle

```diff
@@ -161,7 +161,7 @@ libs += [
   jacksonDataformatCsv: "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:$versions.jackson",
   jacksonModuleScala: "com.fasterxml.jackson.module:jackson-module-scala_$versions.baseScala:$versions.jackson",
   jacksonJDK8Datatypes: "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:$versions.jackson",
-  jacksonAfterburner: "com.fasterxml.jackson.module:jackson-module-afterburner:$versions.jackson",
+  jacksonBlackburn: "com.fasterxml.jackson.module:jackson-module-blackbird:$versions.jackson",
```

Comment:
Can we call it `jacksonBlackbird` instead of `jacksonBlackburn`?

Reply:
Oh, It was a typo! Thanks, @mimaison! I'm gonna push a fix.